src/components/BioMose.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import mose from '../../src/img/mose.png'
+import map from '../../src/img/map.png'
+
+const BioMose = () => {
+  return (
+    <div className="bg-slate-50">
+      <div className="px-6 md:px-16 lg:px-24 pt-8 md:pt-16">
+        <h2 className="text-5xl md:text-6xl mb-3 font-header text-center">About</h2>
```

Comment:
nitpicking (nb): Could we have our designer to check and pick the font size that is available on TailwindCSS for h2 tag in line 9? On figma it's 64px for web screen and 40px for mobile screen which isn't available for us.